src/Donors/resources/admin/components/DonorDetailsPage/Tabs/Overview.tsx

```diff
@@ -1,7 +1,229 @@
+import React from 'react';
+import {__} from '@wordpress/i18n';
+import classnames from 'classnames';
+import StatWidget from '@givewp/src/Admin/components/StatWidget';
+import Header from '@givewp/src/Admin/components/Header';
+import PrivateNote from '@givewp/src/Admin/components/PrivateNote';
+import TimeSeriesChart from '@givewp/src/Admin/components/Charts/TimeSeriesChart';
+import {useDonorStatistics} from '@givewp/donors/hooks/useDonorStatistics';
+import {amountFormatter, formatTimestamp, getRelativeTimeString} from '@givewp/src/Admin/utils';
+import {getDonorOptionsWindowData} from '@givewp/donors/utils';
+import {useDonorDonations} from '@givewp/donors/hooks/useDonorDonations';
+import styles from '@givewp/donors/admin/components/DonorDetailsPage/DonorDetailsPage.module.scss';
+
+/**
+ * @unreleased
+ */
+type Transaction = {
+    campaign: string;
+    status: 'Completed' | 'Pending' | 'Failed' | 'Refunded';
+    timestamp: string;
+    amount: string;
+};
+
+/**
+ * @unreleased
+ */
+const statusMap: Record<string, 'Completed' | 'Failed' | 'Pending' | 'Refunded'> = {
+    publish: 'Completed',
+    completed: 'Completed',
+    processing: 'Pending',
+    pending: 'Pending',
+    failed: 'Failed',
+    cancelled: 'Failed',
+    abandoned: 'Failed',
+    preapproval: 'Pending',
+    revoked: 'Failed',
+    refunded: 'Refunded',
+};
+
+/**
+ * @unreleased
+ */
+const {currency} = getDonorOptionsWindowData();
+
+/**
+ * @unreleased
+ */
 export default function DonorDetailsPageOverviewTab() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const donorId = parseInt(urlParams.get('id') ?? '0');
+    const {statistics: stats, isResolving: statsLoading, hasResolved: statsResolved,} = useDonorStatistics(donorId);
+    const {donations, hasResolved: donationsResolved} = useDonorDonations({donorId});
```

Comment:
Let's add and pull in the give setting payment mode from the window options.  This way if the site is in test mode, we'll see test data.  In the future we may consider adding a test mode toggle but I think this is a quick and effective way to pull it off.

```suggestion
const {currency, mode} = getDonorOptionsWindowData();

/**
 * @unreleased
 */
export default function DonorDetailsPageOverviewTab() {
    const urlParams = new URLSearchParams(window.location.search);
    const donorId = parseInt(urlParams.get('id') ?? '0');
    const {statistics: stats, isResolving: statsLoading, hasResolved: statsResolved,} = useDonorStatistics(donorId, mode);
    const {donations, hasResolved: donationsResolved} = useDonorDonations({donorId, mode});
```